src/organizations/components/OrgProfileTab/CopyableLabeledData.tsx

```diff
@@ -0,0 +1,75 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {
+  FlexBox,
+  ComponentSize,
+  Heading,
+  HeadingElement,
+  FontWeight,
+  JustifyContent,
+} from '@influxdata/clockface'
+import CopyButton from 'src/shared/components/CopyButton'
+
+// Utils
+import {copyToClipboardSuccess} from 'src/shared/copy/notifications'
+import 'src/organizations/components/OrgProfileTab/style.scss'
+
+interface Props {
+  id: string
+  label: string
+  src: string
+  name?: string
+}
+
+const CopyableLabeledData: FC<Props> = ({id, label, src, name}) => {
+  const generateCopyText = (title, text) => () => {
+    return copyToClipboardSuccess(text, title)
```

Comment:
I know this is some copy pasta, but I'm pretty sure this is missing some pieces, like the `dispatch` and `notify`

Reply:
good catch. Twas a lazy copy/pasta. Fixed.